_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,89 @@
+## Android setup
+
+_This step is optional, and can be performed at any time before
+using Flutter to develop apps for Android._
+
+### Install Android Studio
+
+To develop Flutter apps for Android, you need to install Android Studio. After installation,
+you also need to start Android Studio once, to 
```

Comment:
It's in progress, and we're looking for why that's needed.

Any idea why that's needed?